shopping-cart/public/js/shopping-cart.js

```diff
@@ -1,28 +1,28 @@
 $(function(){
-
+  
   var prodbox = {
     app01:$('#app01'),
     ora02:$('#ora02'),
   }
-
+  
   var prod_form  = $('#prod_form')
   var code_input = $('#code_input')
-
+  
   var cart_form  = $('#cart_form')
   var entry_input = $('#entry_input')
-
+  
   for( var code in prodbox ) {
     prodbox[code].click(function(){
       var box = $(this)
       code_input.val(box.attr('id'))
       prod_form.submit()
     })
   }
-
+  
```

Comment:
the file seems to have spaces. Please remove (and set editor to automatically remove)